src/app/features/backoffice/pages/activities-details/activities-details.component.html

```diff
@@ -0,0 +1,77 @@
+<app-header (newItemEvent)="addItem()" ></app-header>
+<div class="detContainer">
```

Comment:
```suggestion
<div class="det-container">
```
es preferible usar esta notacion